### DISCUSSION TITLE
Google TFRC

### DISCUSSION THREAD

**Discussion by 6r1d** [neonbjb/tortoise-tts#127] [General]
Hello and thanks for making Tortoise-TTS. You've mentioned a peculiar part:

> The above points could likely be resolved by scaling up the model and the dataset. For this reason, I am currently withholding details on how I trained the model, pending community feedback.

> If I, a tinkerer with a BS in computer science with a ~$15k computer can build this, then any motivated corporation or state can as well. I would prefer that it be in the open and everyone know the kinds of things ML can do.

Have you considered using TFRC for training to compare your method and more straightforward scaling option you're mentioning?
I believe this amount of resources could help shine the light on what the difference in approaches achieves: https://news.ycombinator.com/item?id=27728225

**Comment by neonbjb** [comment]
A few people have mentioned this to me. The frank answer is I'm not interested in learning Jax (until the world forces me to). Most research these days is done in torch. If I need an implementation of a paper, chances are it is available on github, in torch. By going to Jax+TPUs, I would be forced to rewrite everything. I'm just not interested in doing that.
